target/linux/ath79/dts/ar9344_pcs_cap324.dts

```diff
@@ -150,10 +153,10 @@
 &wmac {
 	status = "okay";
 
+	nvmem-cells = <&macaddr_art_0 (-1)>, <&cal_art_5000>;
```

Comment:
I think the dts file is correct. It seems that wmac always load eeprom from low address `0x1000`. And the PCIe NIC ([AR9382](https://openwrt.org/toh/hwdata/powercloud_systems/powercloud_systems_cap324)) eeprom size should be the same 0x440.

Here is an eeprom collection.
[art.zip](https://github.com/openwrt/openwrt/files/14318197/art.zip)

Reply:
The implication is that currently, both the wmac and pcie devices use the same calibration.

Reply:
I found an older patch can be referenced.
https://patchwork-proxy.ozlabs.org/project/openwrt/patch/1450392667-22656-2-git-send-email-openwrt@daniel.thecshore.com/

Reply:
ping @danielfdickinson

Reply:
Removed mtd-cal-data from pcie instead. It's not correct anyway.